index.html

```diff
@@ -7,10 +7,10 @@ <h1 class="page-heading">EIPs
   <a href="https://gitter.im/ethereum/EIPs?utm_source=badge&amp;utm_medium=badge&amp;utm_campaign=pr-badge"><img src="https://badges.gitter.im/Join%20Chat.svg" alt="Gitter"></a>
   <a href="/last-call.xml"><img src="https://img.shields.io/badge/rss-Last Calls-red.svg" alt="RSS"></a>
 </h1>
-<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.</p>
+<p>Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards. For more information on the EIPs process, see <a href="{{"process"|relative_url}}">Process</a> ({{site.pages|where:"type","Process"|size}}).</p>
```

Comment:
What's the expression at the end for?

Reply:
... I'm honestly not sure. I tried to copy the format for other links, but haven't found any info on how to build this locally to try it out. See this for a similar example: https://github.com/ethereum/EIPs/pull/1932/files/38b0bc1d5f6420c629d006ae8c0c506395052789#diff-eacf331f0ffc35d4b482f1d15a887d3bR32

Reply:
`({{site.pages|where:"type","Process"|size}})` outputs the number of pages with 'type: Process'. I don't think you need it here.

Building locally is just a matter of following the instructions for building a Github Pages Jekyll site.

Reply:
Thanks! Managed to build locally. Will update soon.